pwa/src/translations/en.ts

```diff
@@ -32,4 +32,9 @@ export const en = {
   "No publication date available": "No publication date available",
   "No summary available": "No summary available",
   "Navbar-Logo": "Navbar-Logo",
+  "Opens a new window": "Opens a new window",
```

Comment:
"Opens _in_ a new window" might be better. If you disagree, feel free to not implement.